recipes/pydaily/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pydaily" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 250870fc577b09056f086114fe9c295850675814e566f64ecac92afe6ab602fc
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - deepdish
+    - numpy 
+    - opencv 
+    - pandas
+    - scikit-image
```

Comment:
Only dependencies needed to install the package go here. I am guessing most of these can be removed.

Reply:
I had tried that, but the CI was failing. Anyway, I am trying again.

Reply:
Ahhhh. This line: https://github.com/PingjunChen/pydaily/blob/master/setup.py#L5 That's unfortunate and is not a recommended practice.

cc @conda-forge/core any ideas on what to do here besides including the run deps in host?

If that is the only solution, then we should do that.

Reply:
If we do include the run deps in host, please make a comment as to why so that other maintainers don't have to figure this out.

Reply:
> Ahhhh. This line: https://github.com/PingjunChen/pydaily/blob/master/setup.py#L5 That's unfortunate and is not a recommended practice.

Completely agree. But I don't know how to work around this, at the moment. 

I have reverted this with appropriate comments. Please let me know if another change is necessary.

Reply:
I made a small change so things are readable.